build/container/Makefile

```diff
@@ -1,19 +1,47 @@
 .DEFAULT_GOAL := build/base
 
-INFRASTRUCTURE_WORKSPACE       ?= workspace
-PROJECT_WORKSPACE              ?= $(CURDIR)/$(INFRASTRUCTURE_WORKSPACE)
+INFRASTRUCTURE_WORKSPACE	?= workspace
+PROJECT_WORKSPACE		?= $(CURDIR)/$(INFRASTRUCTURE_WORKSPACE)
 
-TARGET_DIR						= $(PROJECT_WORKSPACE)/target
-NRI_INTEGRATIONS_FILE			= $(CURDIR)/../embed/integrations.version
-NRI_PKG_DIR						?= ${PROJECT_WORKSPACE}/${BASE_OHIS}
-AGENT_BUILD_TARGET_DIR			?= $(CURDIR)/../../target/bin/linux_amd64
+TARGET_DIR			?= $(PROJECT_WORKSPACE)/target/$(GOOS)_$(DOCKER_ARCH)
+NRI_INTEGRATIONS_FILE		= $(CURDIR)/../embed/integrations.version
+NRI_PKG_DIR			?= $(PROJECT_WORKSPACE)/$(BASE_OHIS)
+AGENT_BUILD_TARGET_DIR		?= $(CURDIR)/../../dist
```

Comment:
Can we format this to have it with the same spacing?

Reply:
Weird, in my IDE it looks fine. Shall sort it out 👍🏼

Reply:
I think it's a common issue with tabs. GH renders them 8-char wide, which sometimes breaks the whole thing.

Reply:
spaces vs tabs :D